tools/install_ubuntu_dependencies.sh

```diff
@@ -113,11 +113,20 @@ function install_deadsnakes_ppa(){
     echo ""
     if [[ $REPLY =~ ^[Yy]$ ]]; then
       INSTALL_DEADSNAKES_PPA="yes"
-      export VIRTUAL_ENV_ROOT=".venv"
     fi
   fi
   if [[ "$INSTALL_DEADSNAKES_PPA" == "yes" ]]; then
-    $SUDO apt-get install software-properties-common -y --no-install-recommends
+    $SUDO apt install -f --reinstall python3-minimal
```

Comment:
just glanced over this, but looks pretty janky?

Reply:
yes it was, I had to go away for a bit. it's cleaned up now and I tested the PR with a fresh devcontainer and running `ubuntu_setup.sh`. I successfully got the `openpilot setup done` message.